feature-examples/python/api_key/list_api_key.py

```diff
@@ -0,0 +1,90 @@
+"""Example of using the Vectara REST API to list the API Keys."""
+
+import dataclasses
+import logging
+
+import requests
+
+
```

Comment:
consider adding an exception class for exceptions in this module, for example:
```
class ListApiKeyException(Exception):
    """Base class for exceptions in this module."""
```
No need to add anything more -- the Exception base will handle initialization.

Reply:
As per your other comment, not catching exception in main now.